### PR TITLE
autoscroll navigation and fix subscribe feature

### PR DESCRIPTION
**Pull request by StephenFluin**

### CODE REVIEW

**Comment by StephenFluin** [comment]
@alexroan Can you take a look? <3

File: _src/scroll.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const element = document.querySelector('navigation .active');
```
Comment (alexroan) [inline _src/scroll.js:2]: Instead of auto-scrolling the active menu item to the top of the window, could we scroll it to the centre?

**Review by alexroan — CHANGES_REQUESTED** [review]

File: _src/scroll.js (hunk 1 above)
Comment (StephenFluin) [reply _src/scroll.js:2]: This is what should happen on most browsers: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoViewIfNeeded#browser_compatibility

What browser are you seeing this on?

Comment (alexroan) [reply _src/scroll.js:2]: I'm using Chrome, and the active menu item is being scrolled to the top of the window.

**Comment by StephenFluin** [comment]
@alexroan take another look, flipped the conditional check to fix the bug, and added resizing of the sidebar so it's always the right size.

**Review by alexroan — APPROVED** [review]

**Pull request merged**